api_handlers.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wptdashboard
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/datastore"
+	"net/url"
+)
+
+// apiTestRunsHandler is responsible for emitting test results JSON for all the runs at the given SHA.
+func apiTestRunsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
@lukebjerring @jeffcarp are *very* terse variable names a standard idiom in Go?

Reply:
Yes, for the unavoibable piping of writer and request. Generally we're more informative with our non-boilerplate naming.

Reply:
:+1: